### ISSUE TITLE
chinese summarization

### ISSUE THREAD

**Issue by Tangjiandd** [yaserkl/TransferRL#6]
I use chinese text summarization datasets to train this model.
but when I test the trained model, the result will be "<pad>, <pad>, <pad>, ....,<pad>"

**Comment by ByUnal** [comment]
Yes, I got the similar results when I tried this model for Turkish. I think the model doesn't work for other languages, or we're doing a mistake in tokenization part
